packages/bpk-tokens/src/react-native-output-test.js

```diff
@@ -32,6 +32,8 @@ const ANDROID_ONLY_PROPS = [
   'ELEVATION_XL',
 ];
 const IOS_ONLY_PROPS = [
+  'BORDER_RADIUS_SM',
+  'BORDER_RADIUS_PILL_LG',
```

Comment:
This one should be  `BORDER_RADIUS_PILL` I think!

Reply:
Good catch 👍 I actually just needed to remove both of them from iOS only props - I negotiated with James to consolidate them. 😇